### ISSUE TITLE
VV: Lobbyeinflüsse - Diskussionsprotokoll Discord

### ISSUE THREAD

**Issue by Nalindir** [TVTower/TVTower#127]
Gedankenspiele zum Thema Lobbyeinflüsse

GWRon:
einfach zum auffrischen / "zufaelligermachen" des Spiels

GWRon
Politikskandale die dann die Lobbies beeinflussen - und somit ob man Sendegebiete bekommt etc

GWRon
momentan geht das ja ueberall, aber im code vorgesehen ist, dass Lobbies Anforderungen stellen koennen ... und man sonst keine Sendegenehmigung fuer ein Bundesland erhaelt

GWRon
Lobbies kann man mit entsprechenden Nachrichten und Werbung positiv/negativ beeinflussen

GWRon
man kann die Grenzen ja "gering" halten - halt so, dass mit ein paar Werbeausstrahlungen von "Knarren-joe" die Waffenlobby wieder milde gestimmt ist.

GWRon
vielleicht ists auch schon "too much" (deswegen derzeit alles noch inaktiv)

GWRon
Dazu braucht man ja dann wieder den Statistikbildschirm, Tabellen ,...

Nalindir
muss man mal erlebt haben. grundsätzlich denke ich, wäre das ganz abwechslungsreich. allerdings darf man das auch nicht zu überreizen. gerade waffenlobby zu pushen als tolle errungenschaft für die sendegenehmigung von nrw? schwer vermittelbar im intuitiven spielverständnis. :slight_smile: 
wenn ich im umkehrschluss die waffenlobby loswerden kann, wenn ich sie mitm terroristen aus dem gebäude sprenge und nichts machen muss, kann ich damit leben. :slight_smile:

GWRon
naja es gibt einfach ein paar Lobbies (egal wie man sie nun nennt) und die koennten halt Sachen beeinflussen:
- Werbung ? Noe bekommst Du nicht
- Sendegenehmigung? Hey Freund aus dem Ministerium, der da darf auch mit weniger Senderimage ...

GWRon
Gerade im Austausch fuer andere Anforderungen kann es ganz interessant sein.

GWRon
So koenntest Du als Dauerwerbesender trotzdem neue Sendegebiete erschliessen.

GWRon
Nachteil: Wer mit der Waffenlobby flirtet, hat es schwer in "Friedensliga"-Gebieten :slight_smile:

Nalindir
ja, mir gings dabei mehr um die gewissensfrage. man sollte möglichkeiten haben, das zu umgehen. sei es mehr image, dann können sie sich nicht wehren, oder höhere kosten bei geringem lobby zuspruch. aber kein komplettes aussperren.

GWRon

Lobbies koennten auch "Schwarzer Koffer"-Aktionen freigeben/verhindern (Sabotage etc)

Nalindir
man sollte den spieler nicht nötigen die waffenlobby zu supporten, wenn man sich damit nicht identifzieren kann. :slight_smile:

GWRon
Ja, ein komplettes Aussperren ist doof.

GWRon
Ich mag auch manche werbung nicht aber muss sie senden (da kohle)

GWRon
achso ... ja kann man ja umgehen

GWRon
in dem die Lobby nur Einfluss auf die anderen Anforderungen hat

GWRon
Lobby koennte Preise / Imageanforderungen/ ... beeinflussen

GWRon
Oder Bauzeiten oder aehnliches

GWRon
also "kleine Stellschrauben"

Nalindir
genau, das wäre cool

GWRon
Besonder lukrative Werbung etc - faende ich dennoch nicht soo schlimm.

Nalindir
ja, sowas wird dann besonders interessant, wenn mein balance konzept tatsächlich funktionieren sollte. gerade das expandieren in andere gebiete ist dann im verhältnis zu den betriebskosten extrem teuer. da dann durch lobbyarbeit kosten zu sparen oder eben dickere werbeaufträge erhalten könnte dem spielanteil richtig gewicht geben ohne das balancing in summe zu gefährden.

**Comment by nittka** [comment]
Siehe auch FinishBroadcastingAsProgramme, wo noch der Einfluss der Ausstrahlung auf die Lobbys implementiert werden muss.

**Comment by scr0llbaer9000** [comment]
Ja, das mit den Lobbies war witzigerweise auch in Mad TV ein unfertiges Feature glaub ich. Wollte schon bei "meinen" Filmen da Werte setzen, wenn's etwa um diese typischen Hollywood-Militär-Action-Filme geht, aber hab's erst mal gelassen.

Wär aber schon Aufwand da alle Filme dann diesbezüglich zu evaluieren.

Ein paar neue Lobbies wären auch interessant noch, etwa Christen vs Atheisten, denn Christentum in den US ist ein großes Ding.

**Comment by GWRon** [comment]
Da man ja nicht nur "Christen" hat, waere eine "Fantasiereligion" vielleicht ganz angebracht - da kann man die prima auf die Schippe nehmen :)
Gibt ja Laender mit weit mehr Religionen (und beachtlichen "Personenmengen").

Unverfaenglicher waere es aber, sie "rauszulassen" um nicht Diskussionen zum Thema Christentum, Judentum, Muslime, Buddhismus ... zu ermoeglichen.

**Comment by scr0llbaer9000** [comment]
Für die Christen sind die anderen Religion ebenso Teufelszeug, die würden mit denen keinen Lobbyverband gründen. `;)`

Ich würde es bei einer spezifisch als Christen bezeichneten Lobby lassen. Wir haben ja kulturell bedingt größtenteils westliche Filme im Datenbestand, und ich hab schon viele Kandidaten für ein Christen-vs-Atheisten-Ding auf meiner mentalen Liste, sogar mehr als für das Raucher-/Nichtraucher-Ding (was viel schwieriger sein wird zu recherchieren für jeden Film). Auch "Fantasiereligion" ergäbe hier keinen Sinn.

Atheisten in den US scheinen von anderen Religion weniger getriggert zu werden und scheinen weniger über die herzuziehen. Ist wohl auch ein psychologisches Ding (Traumaverarbeitung, wenn man in einem extrem calvinistischen Umfeld aufgewachsen ist).

Es gibt ein paar wenige Filme, in denen andere Religionen vorkommen, aber die sind selten und wären dann eben "neutral" in Bezug auf diese Lobbies.
